chart/open-feature-operator/README.md

```diff
@@ -19,16 +21,57 @@ The OpenFeature Operator requires [cert manager](https://cert-manager.io/docs/in
 
 ## Install
 
-To install/upgrade the chart with the release name `open-feature-operator`:
+To install the chart with the release name `open-feature-operator`:
+
 ```
-helm upgrade -i open-feature-operator openfeature/open-feature-operator
+helm repo add openfeature https://open-feature.github.io/open-feature-operator/
+helm repo update
+helm upgrade --install open-feature-operator openfeature/open-feature-operator
 ```
+
 This installation will use the default helm configuration, described in the [configuration section](#configuration)
 To overwrite these default values use the `--set` flag when calling `helm upgrade` or `helm install`, for example: 
+
 ```
 helm upgrade -i open-feature-operator ./chart/open-feature-operator --set sidecarConfiguration.port=8080 --set controllerManager.kubeRbacProxy.resources.limits.cpu=400m
 ```
 
+## Upgrade
+
+To install the chart with the release name `open-feature-operator`:
+
+```sh
+helm repo update
+helm upgrade --install open-feature-operator openfeature/open-feature-operator
+```
+
+#### Upgrade CRDs
+
+CRDs are not upgraded automatically with helm (https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
+OpenFeature Operator's CRDs are templated, and can be updated apart from the operator itself by using helm's template functionality and piping the output to `kubectl`:
+
+```sh
+helm template openfeature/open-feature-operator -s templates/{CRD} | kubectl apply -f -
```

Comment:
I took this from the docs @toddbaert wrote last week but I think we may want to more it more clear that this isn't a command that should be run.